spec/duration.html

```diff
@@ -1030,8 +1030,11 @@ <h1>BalanceDurationRelative ( _years_, _months_, _weeks_, _days_, _largestUnit_,
         1. Let _oneYear_ be ! CreateTemporalDuration(_sign_, 0, 0, 0, 0, 0, 0, 0, 0, 0).
         1. Let _oneMonth_ be ! CreateTemporalDuration(0, _sign_, 0, 0, 0, 0, 0, 0, 0, 0).
         1. Let _oneWeek_ be ! CreateTemporalDuration(0, 0, _sign_, 0, 0, 0, 0, 0, 0, 0).
-        1. Set _relativeTo_ to ? ToTemporalDate(_relativeTo_).
-        1. Let _calendar_ be _relativeTo_.[[Calendar]].
+        1. If _relativeTo_ is not *undefined*, then
+          1. Set _relativeTo_ to ? ToTemporalDate(_relativeTo_).
+          1. Let _calendar_ be _relativeTo_.[[Calendar]].
+        1. Else,
+          1. Let _calendar_ be *undefined*.
```

Comment:
I'm not sure how this was unimplementable before; it seems like it would just try to parse the string "undefined" as a date, and throw. In fact, I think this is unimplementable after the change: MoveRelativeDate calls CalendarDateAdd, which asserts that the calendar value is an Object.

Reply:
I think I'm trying to make three points:

1. In the current prose, I'm assuming ToTemporalDate will throw if _relativeTo_ is undefined. As far as I can tell, this means the current prose is implementable, so changing it would not be editorial.
2. I have no opinion on whether the behaviour implied by (1) is desired.
3. The new prose will cause an assertion failure, so if we need a change, this isn't it.

Does that help?

Reply:
Yep. Now that I understand this better, I don't think any change is needed here.